rest_live/consumers.py

```diff
@@ -222,14 +222,20 @@ def model_saved(self, event):
             model = view.get_model_class()
             renderer = view.perform_content_negotiation(view.request)[0]
             is_known_instance = instance_pk in subscription.pks_in_queryset
+            instance = None
 
             # Can't use a get() to check if instance_pk is in the queryset because
             # get_queryset() might return sliced results
-            pks = view.filter_queryset(view.get_queryset()).values_list("pk", flat=True)
-            if instance_pk in pks:
-                instance = model.objects.get(pk=instance_pk)
```

Comment:
I think the cleanest way to do this would be to simply replace this line here with `instance = view.filter_queryset(view.get_queryset()).get(pk=instance_pk) rather than use the for loop.

Reply:
I didn't use get() and opted to use the loop instead because I think using get() won't work for sliced querysets, which was a fix added in #11

Reply:
Makes sense. I am a bit worried about efficiency here. It seems like the `instance_pk in pks` line is already O(n) and this shouldn't make a difference, but I'm pretty sure that getting all the primary keys with `values_list` will never result in more than a single query on an indexed field, while looping through an entire queryset might trigger N+1 queries for any nested objects in the model.

Let's see if there's a specific error when trying to call `get()` on a sliced queryset. If so, we could call `get()`, and only if that returns an error we could fall back to the for-loop.

Reply:
Looks like support for sliced querysets was just removed and the tests were also changed to not check for it 😕